tests/jobs/spearman_correlation_job/test_integration.py

```diff
@@ -38,6 +38,20 @@ def test_main_entry_point_gen_data(monkeypatch):
     # again is a tuple containing the configuration which is a list of configs.
     assert len(run_mock.call_args[0][0]) == 47, "wrong number of clusters"
 
+    cor_matrix_file = (
+        "storage/snake_oil/ensemble/reports/"
+        "SpearmanCorrelationJob/correlation_matrix.json"
+    )
+    with open(cor_matrix_file) as f:
+        json.load(f)
```

Comment:
Can we do any testing besides being valid JSON?

Reply:
Do you know any meaningful tests to the matrix? I am unsure how to calculate it by hand.